python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -181,8 +181,6 @@ def __from_edgelist(
         workers = _client.scheduler_info()["workers"]
         # Repartition to 2 partitions per GPU for memory efficient process
         input_ddf = input_ddf.repartition(npartitions=len(workers) * 2)
-        # FIXME: Make a copy of the input ddf before implicitly altering it.
-        input_ddf = input_ddf.map_partitions(lambda df: df.copy())
```

Comment:
Why did we move this ?

Reply:
That was done when debugging it weeks ago but I can revert it back and make the copy before repartition.

Reply:
I Just didn't see why it would make a difference regarding the bug that's why I left it.

Reply:
Lets move it back and keep it consistent.